plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -1190,6 +1198,26 @@ def get_unique_pkglists(cls, errata_id):
                                                     batchSize=5).next()['pkglists']
         return pkglists
 
+    @classmethod
+    def get_unique_modules(cls, errata_id):
```

Comment:
Why should it be a classmethod?

Reply:
copy-paste design of `get_unique_pkglists` but cool to update